testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeColumnMappingMode.java

```diff
@@ -485,22 +615,12 @@ public void testUnsupportedOperationsColumnMappingMode(String mode)
                 " 'delta.minWriterVersion'='5')");
 
         try {
-            if (!mode.equals("id")) {
-                assertThat(onTrino().executeQuery("INSERT INTO delta.default." + tableName + " VALUES (1, 'one'), (2, 'two')"))
-                        .updatedRowsCountIsEqualTo(2);
-                assertThat(onTrino().executeQuery("DELETE FROM delta.default." + tableName))
-                        .updatedRowsCountIsEqualTo(2);
-                assertThat(onTrino().executeQuery("UPDATE delta.default." + tableName + " SET a_string = 'test'"))
-                        .updatedRowsCountIsEqualTo(0);
-            }
-            else {
-                assertQueryFailure(() -> onTrino().executeQuery("INSERT INTO delta.default." + tableName + " VALUES (1, 'one'), (2, 'two')"))
-                        .hasMessageContaining("Writing with column mapping id is not supported");
-                assertQueryFailure(() -> onTrino().executeQuery("DELETE FROM delta.default." + tableName))
-                        .hasMessageContaining("Writing with column mapping id is not supported");
-                assertQueryFailure(() -> onTrino().executeQuery("UPDATE delta.default." + tableName + " SET a_string = 'test'"))
-                        .hasMessageContaining("Writing with column mapping id is not supported");
-            }
+            assertThat(onTrino().executeQuery("INSERT INTO delta.default." + tableName + " VALUES (1, 'one'), (2, 'two')"))
```

Comment:
As test is named  `testUnsupportedOperationsColumnMappingMode` can we have separate test for write operation and test only unsupported here?

Reply:
Good point. The DML operations here are actually irrelevant now.
I'll remove them.